ee/server/scim/users.go

```diff
@@ -712,6 +712,24 @@ func getConcreteType[T string | bool](u *UserHandler, v interface{}, name string
 	return concreteType, nil
 }
 
+func getConcreteBoolType(u *UserHandler, v interface{}, name string) (bool, error) {
```

Comment:
Is there another code comment somewhere to explain how/why this is being used (i.e. the MS boolean-as-string encoding issue that you mentioned in standup)? If not, would you mind adding something with a link to any supporting documentation for future reference?